README.md

```diff
@@ -48,22 +47,32 @@ var TryStardust = React.createClass({
 module.exports = TryStardust;
 ```
 
-## Develop
+## Installation
+Start by cloning this repo and installing dependencies:
+
+    git clone git@github.com:TechnologyAdvice/stardust.git stardust && cd $_
+    npm install               # or simply npm i
+
+## Usage
+Once dependencies are installed, start the application with:
+
+    npm start
```

Comment:
Let's avoid "Installation" and "Usage" headings in this area.  The instructions here do not relate to installing or using Stardust, but to development.  Something like develop/contributing/etc is better suited.